book/aliases.md

```diff
@@ -28,6 +28,16 @@ Your useable aliases can be seen in `$nu.scope.aliases`.
 
 ## Persisting
 
-To make your alias persistent it must be added to your _config.nu_ file.
+To make your aliases persistent they must be added to your _config.nu_ file by running `config nu` to open an editor and inserting them.
+Then run `exec nu` on linux/mac to reload your nushell or on windows just restart your nushell.
 
-For more details about how to persist aliases so that they're visible when you start up Nushell, see the [configuration chapter](configuration.md).
+## Piping in aliases
+
+Note that `alias uuidgen = uuidgen | tr A-F a-f` (to make uuidgen on mac behave like linux) won't work.
```

Comment:
From a readers perspective I would suggest replacing those two comparitively obscure unix commands with something from nushell's command set or leading in a sentence as not to confuse with command names. (e.g. `tr` wouldn't be an philosophically endorsed nushell solution even though its blazingly fast)

Reply:
this is just the use case I had, which I thought would be a simple example - and the point of it is how to use something outside nu. but i am not familiar with what would be more idiomatic - as I am just starting, so if you have some suggestion I'll just work them in.

Reply:
Agreed. I don't really like the `exec` or the other externals. Although I do think it's important to demonstrate how to use externals.

Reply:
I'll change the example today!

Reply:
looking forward to it @mh-trimble :yum:

Reply:
sorry took me a bit longer - this week was busy. Updated the German version removing the `exec nu` bit and added the more idiomatic example.